pyproject.toml

```diff
@@ -35,6 +35,8 @@ liccheck = "^0.3.12"
 bandit = "^1.6"
 black = {version = "^18.3-alpha.0", allow-prereleases = true}
 flake8 = "^3.7"
+python-freeipa = "^1.0"
```

Comment:
This is not a development dependency, it's already present in the main dependencies.